### PR TITLE
UI tweaks

### PR DESCRIPTION
**Pull request by geoknee**

### CODE REVIEW

File: packages/payment-proxy-client/src/App.tsx [hunk 1]
```diff
@@ -59,11 +59,9 @@ function Copyright(props: any) {
 }
 
 function computePercentagePaid(info: PaymentChannelInfo): number {
-  return (
-    Number(
-      info.Balance.PaidSoFar /
-        (info.Balance.RemainingFunds + info.Balance.PaidSoFar)
-    ) * 100
+  return Number(
+    (100n * info.Balance.PaidSoFar) /
```
Comment (lalexgap) [inline packages/payment-proxy-client/src/App.tsx:63]: I don't think this is correct, since we're only multiplying the numerator by 100?

Comment (lalexgap) [inline packages/payment-proxy-client/src/App.tsx:63]: I think this means that the percentage value we return is "truncated"?
 IE: If there's `65.8941% paid` this function would return `65%`?

Comment (geoknee) [reply packages/payment-proxy-client/src/App.tsx]: I'm going to try out something on this branch which avoids the need for percentages altogether.

File: packages/payment-proxy-client/src/App.tsx [hunk 2]
```diff
@@ -59,6 +59,9 @@ function Copyright(props: any) {
 }
 
 function computePercentagePaid(info: PaymentChannelInfo): number {
+  if (info.Balance.RemainingFunds <= 0) {
```
Comment (lalexgap) [inline packages/payment-proxy-client/src/App.tsx]: Do we need this? The only way we divide by 0 is if both `PaidSoFar` and `RemainingFunds` are `0`

Comment (geoknee) [reply packages/payment-proxy-client/src/App.tsx]: Yes this was wrong, good catch.

**Comment by geoknee** [comment]
Screenshot 2023-09-15 at 23 20 17
Dropping the percentage approach seems to work -- the little tooltip was showing a percentage with the existing method anyway, which is a bit confusing.

File: packages/payment-proxy-client/src/App.tsx [hunk 3]
```diff
@@ -331,24 +323,25 @@ export default function App() {
...
                       valueLabelDisplay="on"
                     />
                     <StorageIcon />{" "}
                     <Typography variant="body2" color="text.secondary">
-                      {paymentChannelInfo?.Balance.Payee}
+                      {paymentChannelInfo?.Balance.PaidSoFar.toString()}
                     </Typography>
                   </Stack>
-                  {paymentChannelId}
+                  {paymentChannelId.slice(0, 6) + "..."}
```
Comment (lalexgap) [inline packages/payment-proxy-client/src/App.tsx:344]: Could be worth throwing this in a helper function; I bet we'll need to shorten channel ids in other places too.

**Review by lalexgap — APPROVED** [review]

**Pull request merged**